boards/arduino-uno/include/periph_conf.h

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (C) 2014 Freie Universität Berlin, Hinnerk van Bruinehsen
+ *               2016 Laurent Navet <laurent.navet@gmail.com>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_arduino-uno
+ * @{
+ *
+ * @file
+ * @brief       Peripheral MCU configuration for the Arduino Uno board
+ *
+ * @author      Hinnerk van Bruinehsen <h.v.bruinehsen@fu-berlin.de>
+ * @author      Laurent Navet <laurent.navet@gmail.com>
+ */
+
+#ifndef PERIPH_CONF_H_
+#define PERIPH_CONF_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   Clock configuration
+ * @{
+ */
+#define CLOCK_CORECLOCK     (16000000L)
+/** @} */
+
+/**
+ * @brief   Timer configuration
+ *
+ * The timer driver only supports the 16-bit timer (Timer1)
+ * so this is the only one we can use here.
+ *
+ * @{
+ */
+#define TIMER_NUMOF         (2U)
+
+#define TIMER_0             MEGA_TIMER1
+#define TIMER_0_MASK        &TIMSK1
+#define TIMER_0_FLAG        &TIFR1
+#define TIMER_0_ISRA        TIMER1_COMPA_vect
+#define TIMER_0_ISRB        TIMER1_COMPB_vect
+/** @} */
+
+/**
+ * @brief   UART configuration
+ * @{
+ */
+#define UART_NUMOF          (1U)
+#define UART_0              MEGA_UART0
+#define UART_0_ISR          USART_RX_vect
+/** @} */
+
+/**
+ * @brief SPI configuration
+ *
+ * The atmega2560 has only one hardware SPI with fixed pin configuration, so all
```

Comment:
@mali yep, it's working now.
But when I try to launch the task-01 from the Tutorials, it always gives me the following error:

> `There are unsatisfied feature requirements: arduino cpp`

Any idea ? Thanks for your answer.